release-please-config.json

```diff
@@ -0,0 +1,30 @@
+{
+  "bootstrap-sha": "6342ecd17b3c26f5d27b33ff49a2ceedc206f140",
+  "exclude-paths": [
+    ".git",
+    ".idea",
+    ".github",
+    ".vscode"
+  ],
+  "release-type": "java",
+  "include-component-in-tag": false,
+  "include-v-in-tag": true,
+  "packages": {
+    ".": {
+      "package-name": "com.microsoft.graph.microsoft-graph",
+      "changelog-path": "CHANGELOG.md",
+      "extra-files": [
+        {
+          "type": "generic",
```

Comment:
we also need to update this
https://github.com/microsoftgraph/msgraph-sdk-java/blob/aa648d1ebe39a670cfe72279de8b6e22152f0c97/src/main/java/com/microsoft/graph/info/Constants.java#L6

And that
https://github.com/microsoftgraph/msgraph-sdk-java/blob/aa648d1ebe39a670cfe72279de8b6e22152f0c97/README.md?plain=1#L23